docs/ide/mcp.md

```diff
@@ -0,0 +1,166 @@
+---
+title: 'Use MCP servers (Preview)'
+description: Adding MCP servers in Visual Studio to extend GitHub Copilot agent capabilities, setting up mcp.json and managing tool permissions.
+ms.date: 5/8/2025
+ms.topic: overview
+author: aayim
+ms.author: meghaanand
+ms.manager: mijacobs
+ms.subservice: ai-tools
+ms.collection: ce-skilling-ai-copilot
+monikerRange: '>= vs-2022'
+---
+# What is MCP?
+
+Model Context Protocol (MCP) is an open standard that enables AI models to interact with external tools and services through a unified interface. In Visual Studio, MCP support enhances GitHub Copilot's agent mode by allowing you to connect any MCP-compatible server to your agentic coding workflow. This article guides you through setting up MCP servers and using tools with agent mode in Visual Studio.
+
+## Prerequisites
++ [Visual Studio 2022 version 17.14](/visualstudio/releases/2022/release-history) and later offer Copilot Agent.
+    + In the initial 17.14 release, you will need to first enable `Agent Mode` in `Tools > Options > GitHub > Copilot`.
++ [Sign in to Visual Studio using a GitHub account](work-with-github-accounts.md) with [Copilot access](https://docs.github.com/en/copilot/about-github-copilot/what-is-github-copilot#getting-access-to-copilot). <br/>
+    + You can use [GitHub Copilot for Free](copilot-free-plan.md). Sign up and leverage AI to code faster and more efficiently.
++ Enable [agent mode in GitHub Copilot Chat](copilot-agent.md).
+
+## How do MCP and Visual Studio extend GitHub Copilot's agent?
+
+* MCP clients (like Visual Studio) connect to MCP servers and request actions on behalf of the AI model
```

Comment:
nit: consider "such as Visual Studio" or omitting this. I misread this initially as, "MCP Clients, similar to what Visual Studio does..."